src/stylesheets/index.css

```diff
@@ -45,27 +45,3 @@
 .link {
   @apply text-navy-700 hover:underline font-semibold;
 }
-
-.video-iframe {
```

Comment:
Refactored this to use Tailwind because I read somewhere that this is what we should be using.